### PR TITLE
51-upload-file-using-translate-client

### PR DESCRIPTION
**Pull request by klavinsjanis**

### CODE REVIEW

File: cmd/translate/main_client_integration_test.go
Comment (VladislavsPerkanuks) [inline cmd/translate/main_client_integration_test.go:42]: Please close and delete the file, to avoid a mess
```go
defer file.Close()
defer os.Remove(file.Name())
```

**Review by VladislavsPerkanuks — COMMENTED** [review]
Before merging into main, you should wait for #52 to be merged, as there are some changes

File: cmd/translate/main_client_integration_test.go
Comment (VladislavsPerkanuks) [reply cmd/translate/main_client_integration_test.go:42]: > It is the caller's responsibility to remove the file when it is no longer needed.

https://pkg.go.dev/os#CreateTemp

Comment (VladislavsPerkanuks) [reply cmd/translate/main_client_integration_test.go:42]: I guess you are right, `t.TempDir` gets removed, after tests :)

File: cmd/client/cmd/service.go
Comment (janishorsts) [inline cmd/client/cmd/service.go:12]: Let's use this one instead https://github.com/rodaine/table . It's simpler and more compact output.

File: cmd/client/cmd/service.go
Comment (janishorsts) [inline cmd/client/cmd/service.go:104]:
```suggestion
				return fmt.Errorf("upload file: get cli parameter 'timeout': %w", err)
```

File: cmd/client/cmd/service.go
Comment (janishorsts) [inline cmd/client/cmd/service.go:117]:
```suggestion
				return fmt.Errorf("upload file: get cli parameter 'language': %w", err)
```

File: cmd/client/cmd/service.go
Comment (janishorsts) [inline cmd/client/cmd/service.go:150]:
```suggestion
	uploadFlags.StringP("file", "f", "", "file path")
```

**Review by janishorsts — APPROVED** [review]
Replace the table package. Everything else is a minor issue. And has a very high probability of changing in the future.

**Pull request closed without merging**
